src/Microsoft.DotNet.AsmDiff/DiffCSharpWriter.cs

```diff
@@ -193,25 +193,38 @@ public override void Visit(IEnumerable<MemberMapping> members)
 
         public override void Visit(MemberMapping member)
         {
-            IDisposable style = null;
-
-            if (this.HighlightBaseMembers)
+            if (!IsPropertyOrEventAccessor(member.Representative))
             {
-                if (member.Representative.IsInterfaceImplementation())
-                    style = _syntaxWriter.StartStyle(SyntaxStyle.InterfaceMember);
-                else if (member.Representative.IsOverride())
-                    style = _syntaxWriter.StartStyle(SyntaxStyle.InheritedMember);
-            }
+                IDisposable style = null;
 
-            WriteHeader(member);
-            if (style != null)
-                style.Dispose();
+                if (this.HighlightBaseMembers)
+                {
+                    if (member.Representative.IsInterfaceImplementation())
+                        style = _syntaxWriter.StartStyle(SyntaxStyle.InterfaceMember);
+                    else if (member.Representative.IsOverride())
+                        style = _syntaxWriter.StartStyle(SyntaxStyle.InheritedMember);
+                }
+
+                WriteHeader(member);
+                if (style != null)
+                    style.Dispose();
```

Comment:
This could be simplified to just:
```suggestion
                    style?.Dispose();
```